packages/ui/components/ui/copy-to-clipboard.tsx

```diff
@@ -19,7 +19,11 @@ const CopyToClipboard = React.forwardRef<HTMLButtonElement, CopyToClipboardProps
 
     return (
       <Button
-        className={cn(copied && 'cursor-default text-teal-500 hover:no-underline', className)}
+        className={cn(
+          copied && 'cursor-default text-teal-500 hover:no-underline',
+          'block',
```

Comment:
Added the `block` class to remove the weird spacing beneath it that sometimes happens when it's displayed inline. This helps with when the button is displayed in a flex row.